trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/index.test.ts

```diff
@@ -0,0 +1,27 @@
+import httpMocks from 'node-mocks-http';
+import { aTfmSessionUser } from '../../../../../test-helpers';
+import { createRecordCorrectionRequest, CreateRecordCorrectionRequestRequest } from '.';
+
+console.error = jest.fn();
+
+describe('controllers/utilisation-reports/record-corrections/create-record-correction-request', () => {
+  const userToken = 'user-token';
+  const requestSession = {
+    userToken,
+    user: aTfmSessionUser(),
+  };
+
+  it('should render create record correction request page', () => {
+    // Arrange
+    const { req, res } = httpMocks.createMocks<CreateRecordCorrectionRequestRequest>({
+      session: requestSession,
+      params: { reportId: '123', feeRecordId: '456' },
+    });
+
```

Comment:
There's no mocking at present as the data is hardcoded - this will be changed in a subsequent PR, and this test file will be updated